### ISSUE TITLE
Cannot install Django 1.11.13  with Python 2.7.15

### ISSUE THREAD

**Issue by u9235044** [pypa/pipenv#2285]
Hi there,

I am trying to use heroku and face issue during the setup because I cannot install Django with pipenv.
Here I found a similar issue before but the solution is not working for me, so I think it's another issue with pipenv.

The issue I found is here:
https://github.com/pypa/pipenv/issues/1838

Please run `$ python -m pipenv.help`, and paste the results here.
https://pastebin.com/dPZ7D9gG

------------

##### Expected result

Install Django 1.11.13 with Python 2.7.15

##### Actual result

An error occurred while installing django==2.0.5! Will try again.

##### Steps to replicate
1. Install pipenv
My pipenv version is 2018.5.18

2.. Get the code from heroku.
git clone https://github.com/heroku/python-getting-started.git
cd python-getting-started

3. Modify the Pipfile on [requires] field
Original: python_version = "3.6"
After: python_full_version = "2.7.15"
(I tried python_version ="2.7" but not working, too.)

4. trying to install
pipenv --two
pipenv install
(or pipenv install --system --deploy)

**Comment by u9235044** [comment]
Hi there,

Thanks for reply :)
What you mean fixed in master? Does it means this will fix in the next release?

**Comment by techalchemy** [comment]
Yep exactly, it was quite annoying as I am actually in the same situation as you at work — django <2 with python 2.7 and using pipenv for deployments

**Comment by u9235044** [comment]
Got it, thanks!!!  :)
